### DISCUSSION TITLE
how to pause the other while one is playing wavesurfer.js?

### DISCUSSION THREAD

**Discussion by Gayrat-SE** [katspaugh/wavesurfer.js#3096] [Q&A]
how to pause the other while one is playing wavesurfer.js?

**Comment by katspaugh** [comment]
As they say on StackOverflow, what have you tried?

**Reply by Gayrat-SE** [reply]
function disable_audio() {
  document.addEventListener('play', function (e) {
    if (e.target.id !== 'sound1') { // не отключать звук SMS
      if (!e.target.classList.contains('waveaudio')) {
        const audios = document.querySelectorAll('.audio_block .waveaudio');
        audios.forEach((audio) => {
          if (audio !== e.target) {
            audio.pause();
          }
        })
      }
    }
  }, true);
}

**Reply by Gayrat-SE** [reply]
I tried to make like this but it didn't work

**Reply by katspaugh** [reply]
The document object doesn't emit a play event. Only audio/video elements do.
You should listen to it on all of them, and when one of them starts playing, pause the others.

**Reply by Gayrat-SE** [reply]
is there any example you have? can you share?

**Reply by katspaugh** [reply]
I do not, try chatgpt.
